src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -165,7 +167,36 @@ def create(dataset, session_id, target, features=None, prediction_window=100,
     dataset = _tkutl._toolkits_select_columns(dataset, features + [session_id, target])
     _tkutl._raise_error_if_sarray_not_expected_dtype(dataset[target], target, [str, int])
     _tkutl._raise_error_if_sarray_not_expected_dtype(dataset[session_id], session_id, [str, int])
+    params = {
+        'use_tensorflow': False
+        }
 
+    if '_advanced_parameters' in kwargs:
+        # Make sure no additional parameters are provided
+        new_keys = set(kwargs['_advanced_parameters'].keys())
+        set_keys = set(params.keys())
+        unsupported = new_keys - set_keys
+        if unsupported:
+            raise _ToolkitError('Unknown advanced parameters: {}'.format(unsupported))
+    params.update(kwargs['_advanced_parameters'])
+
+    if params['use_tensorflow'] :
+
+        name = 'activity_classifier'
+
+        # Imports tensorflow
+        import turicreate.toolkits.libtctensorflow
+        from ._tf_model_architecture import ActivityTensorFlowModel
+
+
+        model = _turicreate.extensions.activity_classifier()
+        options = {}
```

Comment:
What about `features`?

Reply:
passing it in the dataset, right?

Reply:
The options dictionary I made to sent the arguments needed for init_options. There aren't any features. I am sending the features when I call train right?